### ISSUE TITLE
Disable pointer-events by default?

### ISSUE THREAD

**Issue by annapowellsmith** [kiln/flourish-popup#6]
In both the templates where I've imported this plugin and called `popup.show()` on mouseover, I've had to add:

    .flourish-popup {
       pointer-events: none;
    }

to stop the popups bouncing furiously. The first time, I spent quite a bit of time trying to debug the issue, since it didn't occur to me that `pointer-events` wouldn't already be set in the plugin.

I can see that we don't have `pointer-events: none` set by default because we're supporting click events. But I think it would be better if the plugin didn't need tweaks by default.

**Comment by theduncanclark** [comment]
Yes, agree. You, me and even @robinhouston (who made the library!) have all been caught out by this. Also based on our experience so far we've used the library far more for non-clickable popups than clickable. So I would argue for non-clickable by default.

But maybe changing the default on `.flourish-popup` is the wrong solution because if you *do* want them clickable then you still don't want the flickering effect. So whatever else we do maybe we need to try killing the flicker by turning off pointer-events for the SVG that provides the drop shadow.

**Comment by robinhouston** [comment]
> turning off pointer-events for the SVG that provides the drop shadow

@theduncanclark Are you thinking only of the situation where the popup is pointed at the cursor position?

**Comment by theduncanclark** [comment]
> @theduncanclark Are you thinking only of the situation where the popup is pointed at the cursor position?

I was just speculating that having pointer events on the dropshadow makes the flicker effect worse and harder to understand (as it's not obvious whether you've moused over the popup), though it certainly wouldn't solve the whole problem.

**Comment by animateddata** [comment]
This has caught me out too :) - the popup was only updating/moving once the cursor had moved a sufficient distance away.

**Comment by micycle** [comment]
cf https://github.com/kiln/flourish/pull/1198

**Comment by robinhouston** [comment]
I suspect I have a minority view here, but I think the correct approach is change the documentation to emphasise this potential problem, rather than setting opinionated defaults that assume the popups are being used in a particular way.

**Comment by timbrock** [comment]
Could we add a `pointerEvents` method too? It would be nice to set in JS when instantiating a popup. And you could explain the problem and default value in the README for the pointerEvents method.

**Comment by robinhouston** [comment]
Okay. So, I agree we should set pointer-events: none on the drop shadow. Sorry for not being clearer about that (in my own mind, as well as in my comments).
